[PTFE] New admin user workflow

koikonom opened the pull request.

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
-Administrator account. You will be able to create additional administrators
-later.
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
```
> After clicking "Open" below the button labeled "Stop Now", you'll see a page where you can create the first PTFE admin user.

armchairlinguist replied:
As a side note - do we have any control over what the button says? "Open" is pretty vague.

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 
 ### Creating an Organization
 
 The next step will create the first Organization.
```
Our general style is not to uppercase entity types - can we change this to "first organization" while we're in here?

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 ![Create a new organization](./assets/create-organization.png)
 
-If you skip this step, you will be prompted the create the first organization
-when you first log in.
```
Does this not apply anymore?

koikonom replied:
There is no skip button. If you navigate elsewhere (user prefs, admin settings, whatever) and then click back on the TFE icon you'll be asked to create an org. If you log out and log in again, you'll be asked to create an org.

armchairlinguist replied:
Oh, good call. I forgot that you're already logged in with this new flow.

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
-When you have added your configuration press "Save and Continue," or skip to the
-next step.
+After this is done, you can either continue with the creation of a new workspace,
+choose to configure other aspects of PTFE, or add more users.
```
I might change this to explicitly include a mention of "configure login settings such as SAML or Twilio", with links to the relevant configuration pages. (If we don't have existing details for Twilio, we'll need them. :)

koikonom commented:
@armchairlinguist addressed most comments apart from the last one about twilio, because I had some questions but no time to ask :)

I am pushing this just in case it needs to go out while I am away. Please feel free to change as required, otherwise we can talk about it on Monday.

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
-### Configure SMTP (optional)
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
+You will be able to create additional administrators once you login.
```
once you login. > once you log in.

armchairlinguist commented on content/source/docs/enterprise/private/config.html.md:
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
-Navigate to `https://<TFE HOSTNAME>/admin/bootstrap` in your browser. You will
-be presented with a status page that looks like this:
+Navigate to `https://<TFE HOSTNAME>:8800/` in your browser. You will
+be presented with the installer's dashboard:
```
installer's > installer

koikonom commented:
No worries that's why I keep asking you for feedback. I'd expect nothing less :P
